src/java/picard/analysis/CollectWgsMetrics.java

```diff
@@ -209,17 +224,29 @@ protected int doWork() {
             baseQHisto.increment(i, baseQHistogramArray[i]);
         }
 
-        final WgsMetrics metrics = generateWgsMetrics();
-        metrics.GENOME_TERRITORY = (long) histo.getSumOfValues();
-        metrics.MEAN_COVERAGE = histo.getMean();
-        metrics.SD_COVERAGE = histo.getStandardDeviation();
-        metrics.MEDIAN_COVERAGE = histo.getMedian();
-        metrics.MAD_COVERAGE = histo.getMedianAbsoluteDeviation();
+        // Construct and write the outputs
+        final Histogram<Integer> baseQHetHisto = new Histogram<Integer>("value", "baseq_count");
+        for (int i=0; i<baseQHetSensHistogram.length; ++i) {
+            if(i>0 && i<17) {
+                baseQHetHisto.increment(0, baseQHetSensHistogram[i]);
+                baseQHetHisto.increment(i, 0);
+            }
+            else {
+                baseQHetHisto.increment(i, baseQHetSensHistogram[i]);
```

Comment:
Actually, you can replace this entire `if. . . else` by a single line: `baseQHetHisto.increment( i < 17 ? 0 : i, baseQHetSensHistogram[i])`.